src/lib/core/ripple/ripple-renderer.ts

```diff
@@ -211,15 +205,17 @@ export class RippleRenderer {
     const rippleEl = rippleRef.element;
     const animationConfig = {...defaultRippleAnimationConfig, ...rippleRef.config.animation};
 
+    // This starts the fade-out transition and will fire the transition end listener that
+    // removes the ripple element from the DOM.
     rippleEl.style.transitionDuration = `${animationConfig.exitDuration}ms`;
     rippleEl.style.opacity = '0';
     rippleRef.state = RippleState.FADING_OUT;
 
-    // Once the ripple faded out, the ripple can be safely removed from the DOM.
-    this.runTimeoutOutsideZone(() => {
```

Comment:
In the Chromium issue, someone mentioned that using `requestAnimationFrame` could work around the behavior- would it work in our case to just add a `requestAnimationFrame` at the end of the timeout (also possibly reducing the timeout duration slightly)

Reply:
Why should we do that? Since we run an animation through CSS, I feel like using `transitionend` is the way to go. We'd save multiple (nested) macrotasks and also have the code more clean and reasonable.

Reply:
Just thinking about different options (this one for a more minimal change).